examples/authProvider/googleLogin/src/interfaces/google.d.ts

```diff
@@ -3,15 +3,15 @@ interface IdConfiguration {
     auto_select?: boolean;
     callback: (handleCredentialResponse: CredentialResponse) => void;
     login_uri?: string;
-    native_callback?: (...args: any[]) => void;
+    native_callback?: (...args: any[]) => void; // eslint-disable-line
     cancel_on_tap_outside?: boolean;
     prompt_parent_id?: string;
     nonce?: string;
     context?: string;
     state_cookie_domain?: string;
     ux_mode?: "popup" | "redirect";
     allowed_parent_origin?: string | string[];
-    intermediate_iframe_close_callback?: (...args: any[]) => void;
+    intermediate_iframe_close_callback?: (...args: any[]) => void; // eslint-disable-line
```

Comment:
```suggestion
    intermediate_iframe_close_callback?: (...args: unknown[]) => void;
```

Looks like its going to be enough to fix the `any` with `unknown`

Reply:
done